app/views/links/show.html.erb

```diff
@@ -15,16 +27,6 @@
   <% end %>
 <% end %>
 
-<% if @link.shares.any? %>
-  <h2 class="text-4xl">Shares</h2>
-
-  <ul class="link-shares">
-  <% @link.shares.find_each do |share| %>
-    <li class="<%= cycle("bg-slate-300", "bg-red-100") %>"><%= render share %><%= link_to "Clone", link_clone_path(@link, share) %></li>
-  <% end %>
-  </ul>
-<% end %>
-
 <div class="mb-6">
   <%= link_to "Back to links", links_path %> |
   <%= link_to "Share", new_link_share_path(@link) %>
```

Comment:
maybe you can move this all to the top right before the list of shares? now if you go to an item for the first time you have the `Share` at the bottom and then the list appears at the top, so you have to keep going from top to bottom

I would move these two link to the top so also if you want to go back to the links you don't have to scroll all the way down

Reply:
You mean something like this?:
<img width="1019" alt="Screen Shot 2023-05-15 at 4 00 39 PM" src="https://github.com/fastruby/librarian/assets/6892410/1e22b919-5180-4f2b-b0e3-30a0fa43bca3">

Reply:
yeah something like that, I don't know if we need to move the `Back to links` link there though, I think that one does make sense at the bottom to not have to scroll all the way up (we already have the navbar to go back when you are at the top)

maybe change the `Share` to `New Share Link` or something like that? just `Share` is confusing to me